src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -248,7 +248,13 @@ private void OnConnectionChanged(object sender, bool connected)
 
                     // s_lastRemoteClientTask info should be saved in the dump
                     // report NFW when connection is closed unless it is proper shutdown
-                    FatalError.ReportWithoutCrash(new Exception("Connection to remote host closed"));
```

Comment:
❓ Would it break bucketing to modify the implementation of `ReportWithoutCrash`?